index.html

```diff
@@ -58,13 +57,13 @@ <h1>My Recent Works</h1>
       <hr />
     </div>
 
-    <img class="middle-image" src="assets/images/ImgPlaceholderu.png" alt="" />
+    <img class="middle-image" src="assets/images/ImgPlaceholderu.png" alt="Image place holders " />
   </div>
 
 
   <section class="middle-section">
     <div class="middle-image1-container">
-      <img class="middle-image1" src="assets/images/bigimageplaceholder.png" alt="">
+      <img class="middle-image1" src="assets/images/bigimageplaceholder.png" alt="Image place holders">
       <div class="middle-1-container">
         <h1>Multi-Post Stories</h1>
```

Comment:
- [ ] Kindly change all `h1` tags except the first one in the headline section to `h2/h3...`. It's a good practice for a webpage to have only one `h1` tag and all subsequent headings can be h2, h3 in a hierarchical order.  For this project and for hierarchy purposes, you should have only one h1 tag on your entire page. please change all the headings for your sections to the `h2` tag 👍